core/urls.py

```diff
@@ -20,5 +20,5 @@
 
 urlpatterns = [
     path("admin/", admin.site.urls),
-    path("api/v1/auth/", include("users.urls")),
+    path("api/v1/", include("users.urls")),
```

Comment:
Create an app called app which will house all your apps, and then point this path to the app.urls.py which will then have the paths for the different paths